validator/submission_tester/api.py

```diff
@@ -145,22 +92,11 @@ async def stream_logs(websocket: WebSocket):
 
     await websocket.send_json({"version": API_VERSION})
 
-    loop = asyncio.get_running_loop()
-
-    old_out = sys.stdout
-    old_err = sys.stderr
-
-    out = WebSocketLogStream(websocket, "out", sys.stdout, loop)
-    err = WebSocketLogStream(websocket, "err", sys.stderr, loop)
-
     try:
-        sys.stdout = out
-        sys.stderr = err
-
         while True:
-            await asyncio.sleep(0.5)
-
-            await asyncio.gather(out, err)
-    finally:
-        sys.stdout = old_out
-        sys.stderr = old_err
+            if not logs.empty():
+                message = logs.get()
+                await websocket.send_text(f"[API] - {message}")
```

Comment:
That's gonna look like `[API] - out: {message}`, so the validator won't know how to parse it